receiver/prometheusreceiver/metrics_receiver_protobuf_test.go

```diff
@@ -257,12 +282,49 @@ func TestNativeVsClassicHistogramScrapeViaProtobuf(t *testing.T) {
 						{Offset: 2, Length: 1},
 					},
 					PositiveDelta: []int64{1, 0},
+					Exemplars: []*dto.Exemplar{
+						{
+							Value:     0.8,
+							Label:     []dto.LabelPair{{Name: "tnh_e1", Value: "mv1"}},
+							Timestamp: &types.Timestamp{Seconds: 234, Nanos: 456},
+						},
+						{
+							Value:     111,
+							Label:     []dto.LabelPair{{Name: "tnh_e2", Value: "mv2"}},
+							Timestamp: &types.Timestamp{Seconds: 432, Nanos: 456},
+						},
+					},
 				},
 			},
 		},
 	}
 	prometheusMetricFamilyToProtoBuf(t, buffer, nativeHistogram)
 
+	// Exemplar check function to check that the right values go to the
+	// right datapoints.
+	tchExemplarCheck := func(t *testing.T, hdp pmetric.HistogramDataPoint) {
+		require.Equal(t, 2, hdp.Exemplars().Len())
+		values := []float64{hdp.Exemplars().At(0).DoubleValue(), hdp.Exemplars().At(1).DoubleValue()}
+		require.Contains(t, values, 8.1)
+		require.Contains(t, values, 18.0)
+	}
+	tmhcExemplarCheck := func(t *testing.T, hdp pmetric.HistogramDataPoint) {
+		require.Equal(t, 1, hdp.Exemplars().Len())
+		require.Equal(t, 0.1, hdp.Exemplars().At(0).DoubleValue())
+	}
+	tmhnExemplarCheck := func(t *testing.T, hdp pmetric.ExponentialHistogramDataPoint) {
```

Comment:
```suggestion
	tmnhExemplarCheck := func(t *testing.T, hdp pmetric.ExponentialHistogramDataPoint) {
```

I'm not a fan of those super shortened variable names, but was this supposed to represent `testmixednativehistogram`?

Reply:
done